ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -219,6 +350,20 @@ private void writeHeader( HttpServletResponse response )
         }
     }
 
+	public String getIpAddress()
```

Comment:
What works for me is to initially try and guess a default IP address, but be able to use an openfire property to specify whatever the network people assign to me